doc/bib/elephant.bib

```diff
@@ -64,6 +83,18 @@ @article{Riehle97_1950
   doi = {10.1126/science.278.5345.1950}
 }
 
+@article{seth2007granger,
+  author = {Seth, A. },
+  title   = {Granger causality},
+  year    = {2007},
+  journal = {Scholarpedia},
+  volume  = {2},
+  number  = {7},
+  pages   = {1667},
+  doi     = {10.4249/scholarpedia.1667},
+  note    = {revision \#127333}
```

Comment:
I think revision can go out, at least the DOI should be clear to which revision it points.

Reply:
Revision is not reflected in the DOI (if that is what you are referring to?):
[Revision #127333 http://dx.doi.org/10.4249/scholarpedia.1667](http://www.scholarpedia.org/article/Granger_causality)
[Revision #91329 http://dx.doi.org/10.4249/scholarpedia.1667](http://www.scholarpedia.org/w/index.php?title=Granger_causality&oldid=91329)